lib/Traits/TimeVaryingTraits.ts

```diff
@@ -12,13 +12,7 @@ export default class TimeVaryingTraits extends mixTraits(CatalogMemberTraits) {
 
   @primitiveTrait({
     name: "Initial Time Source",
-    description:
-      "The initial time to use if `Current Time` is not specified. Valid values are:\n\n" +
-      "  * `start` - the dataset's start time\n" +
-      "  * `stop` - the dataset's stop time\n" +
-      "  * `now` - the current system time. If the system time is after `Stop Time`, `Stop Time` is used. If the system time is before `Start Time`, `Start Time` is used.\n\n" +
-      "  * `none` - do not automatically set an initial time\n" +
-      "This value is ignored if `Current Time` is specified",
+    description: `The initial time to use if \`Current Time\` is not specified. Valid values are <br/> * \`start\` - the dataset's start time <br/> * \`stop\` - the dataset's stop time <br/> * \`now\` - the current system time. If the system time is after \`Stop Time\`, \`Stop Time\` is used. If the system time is before \`Start Time\`, \`Start Time\` is used. <br/> * \`none\` - do not automatically set an initial time <br/> This value is ignored if \`Current Time\` is specified.`,
```

Comment:
This one should be on multiple lines too.

Reply:
Done